clients/cli/Cargo.toml

```diff
@@ -18,3 +18,4 @@ chrono = "0.4"
 qr2term = "=0.2.1"
```

Comment:
checkout: https://github.com/lockbook/lockbook/blob/master/docs/CONTRIBUTING.md for a handful of pr heading and description conventions